glide.go

```diff
@@ -639,3 +649,42 @@ func glidefile(c *cli.Context) string {
 	}
 	return a
 }
+
+const (
+	textFormat       = "text"
+	jsonFormat       = "json"
+	jsonPrettyFormat = "json-pretty"
+)
+
+func outputList(l action.PackageList, format string) {
```

Comment:
The `glide.go` file is already fairly long. Can we move `outputList` into `action/list.go`.  Maybe instead of `action.List` returning something it takes in the format switch and handles the display. That's how the other commands work.